site/src/pages/components/invokers.explainer.mdx

```diff
@@ -1,10 +1,17 @@
 ---
-menu: Active Proposals
+menu: Graduated Proposals
```

Comment:
Will this change the URL? I assume not, but I want to make sure because we link to these explainers all over the place.

Reply:
No it's only used for the sidebar links. The file name determines the URL but good to be sure